docs/sdk/server-side-sdks/dotnet-local/dotnet-local-gettingstarted.md

```diff
@@ -24,10 +24,52 @@ namespace Example
     {
         static Main(string[] args)
         {
-            DVCLocalClientBuilder apiBuilder = new DVCLocalClientBuilder();
-            using DVCLocalClient api = apiBuilder.SetSDKKey("<DVC_SERVER_SDK_KEY>")
-                      .Build();
+            DVCLocalClientBuilder clientBuilder = new DVCLocalClientBuilder();
+            using DVCLocalClient client = clientBuilder.SetSDKKey("<DVC_SERVER_SDK_KEY>").Build();
```

Comment:
We should set a best practice of sourcing this from an env var even in our docs.

Reply:
agreed but I think we can catch that on the renaming work Jonathan is doing since we have to go in and rework these again anyway to used "DEVCYCLE" instead of "DVC"